orders/src/main/java/local/naught/orders/model/Agents.java

```diff
@@ -0,0 +1,127 @@
+package local.naught.orders.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "agents")
+public class Agents
+{
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long agentcode;
+    private String agentname;
+    private String workingarea;
+    private double commission;
+    private String phone;
+    private String country;
+
+    @OneToMany(mappedBy = "agent",
+            cascade = CascadeType.ALL,
+            orphanRemoval = true)
+    @JsonIgnore
+    private List<Customers> customers = new ArrayList<>();
+
+    @OneToMany(mappedBy = "agent",
+            cascade = CascadeType.ALL,
+            orphanRemoval = true)
+    @JsonIgnore
+    private List<Orders> orders = new ArrayList<>();
```

Comment:
Line 32 - Don't rely on `@JsonIgnore` - it will prevent printing of an agent's order list if you print some agent object(s) directly.

Instead, use `@JsonIgnoreProperties`, and pass in String values representing the order object's properties `({"agent", "customer"})`.